apis/console/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use super::*;
+use core::fmt::Write;
+use libtock_platform::ErrorCode;
+use libtock_unittest::{command_return, fake, ExpectedSyscall};
+
+type Console = super::Console<fake::Syscalls>;
+
+#[test]
+fn no_driver() {
+    let _kernel = fake::Kernel::new();
+    assert!(!Console::driver_check());
+}
+
+#[test]
+fn driver_check() {
+    let kernel = fake::Kernel::new();
+    let driver = fake::Console::new();
+    kernel.add_driver(&driver);
+
+    assert!(Console::driver_check());
+    assert_eq!(driver.take_bytes(), &[]);
+}
+
+#[test]
+fn write_bytes_all() {
+    let kernel = fake::Kernel::new();
+    let driver = fake::Console::new();
+    kernel.add_driver(&driver);
+
+    Console::write_all("foo".as_bytes()).unwrap();
```

Comment:
IIRC you can use `b"foo"` instead of `"foo".as_bytes()`.

Reply:
Changed.